apps/application-system/api/infra/application-system-api.ts

```diff
@@ -270,6 +263,7 @@ export const serviceSetup = (services: {
       DirectorateOfImmigration,
       SocialInsuranceAdministration,
       OccupationalLicenses,
+      WorkMachines,
```

Comment:
~This seems unrelated to the topic of the PR?

Reply:
I think somebody else forgot to add this with their changes, I had to add this in order for the feature deploymenet to successfully deploy.